### ISSUE TITLE
GBA freezes in game

### ISSUE THREAD

**Issue by Colypoly** [OpenEmu/OpenEmu#2515]
For some reason when I play megaman battle network 5, at a certain point in the game it just freezes and although I can restart and get out, It won't let me beyond a certain point. There are two versions of the game (protoman and colonel), and each of them are at the same point and both won't let me progress. This doesn't have to do with time because I've played colonel for 6 hours and protoman for 1. I don't know if it will effect my other games but I would like this to be fixed. It only freezes when I "jack in" to the fourth box in the abandoned mining facility. I am running osx 10.11.4, el captain and my openemu is version 2.0.1. I have no clue what I'm supposed to do.

**Comment by clobber** [comment]
Sounds like a bug with VBA. We don't develop the emulators so this is not really something we can debug.

I'd recommend transferring your save file and using mGBA standalone for now (https://mgba.io/) until we have switched over to that emulator as our new default for GBA games in a future release.

**Comment by Colypoly** [comment]
two questions:
1) How do I transfer the save files to the mGBA emulator from OpenEmu?
and...
2) Is there any timeframe for when the mGBA is replacing VBA? I really love OpenEmu and it sucks to have to switch over.
@clobber Thanks a lot for helping me out I really appreciate it. I've tested it and it works but I still need question 1 answered because I don't want to start all my games over again that I played using OpenEmu
